config/core/test-configs.yaml

```diff
@@ -2242,10 +2255,12 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-alsa
 
   - device_type: minnowboard-turbot-E3826
     test_plans:
       - baseline
+      - kselftest-alsa
```

Comment:
There isn't enough capacity in the Collabora lab for running anything else than baseline on this platform, partly because other projects run a lot of tests on them. So please drop this line too.

Reply:
Shouldn't this be restricted in the lab configuration? It does feel like we've got the lab and board configurations too tightly wound here...